.changes/v2.15.0/436-improvements.md

```diff
@@ -0,0 +1 @@
+* Disk methods have now the ability to access new properties from API version 36.0. They are: `DiskRecordType.SharingType`, `DiskRecordType.UUID`, `DiskRecordType.Encrypted`, `Disk.SharingType`, `Disk.UUID` and `Disk.Encrypted`. Also, new method added `Disk.GetAttachedVmsHrefs`. `Vms.VmReference` changed to array to fix not correct deserialization [GH-436]
```

Comment:
IMO the `Vms.VmReference changed to array to fix not correct deserialization` is also worth a `bugfix` entry for `VmReference`field. That automatically explains why a breaking change is done in structure + is easier to spot for someone looking bugfixes.

Reply:
moved to separate changelog bug fixes file